playground/images.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+
+export function HomeMajorMonotone() {
```

Comment:
Nitpick: @kaelig probably has some thoughts on how some of these can be simplified - you probably don't need the`g`/`defs` stuff, or width/height delcarations on the svg files